doc/sdh/build/cmd_build.adoc

```diff
@@ -0,0 +1,155 @@
+[[syndesis-build]]
+## `syndesis build`
+
+The main goal of `syndesis` is to build Syndesis.
```

Comment:
Perhaps just to point that the `syndesis` script is in `tools/bin` and that it can be added to $PATH?